[2025-01-23T03:59:37.146+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: All_Layer.clean_credit manual__2025-01-23T03:54:02.576089+00:00 [queued]>
[2025-01-23T03:59:37.159+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: All_Layer.clean_credit manual__2025-01-23T03:54:02.576089+00:00 [queued]>
[2025-01-23T03:59:37.160+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2025-01-23T03:59:37.176+0000] {taskinstance.py:1382} INFO - Executing <Task(SparkSubmitOperator): clean_credit> on 2025-01-23 03:54:02.576089+00:00
[2025-01-23T03:59:37.182+0000] {standard_task_runner.py:57} INFO - Started process 10629 to run task
[2025-01-23T03:59:37.185+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'All_Layer', 'clean_credit', 'manual__2025-01-23T03:54:02.576089+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/All_Offi.py', '--cfg-path', '/tmp/tmpln842r6k']
[2025-01-23T03:59:37.188+0000] {standard_task_runner.py:85} INFO - Job 62: Subtask clean_credit
[2025-01-23T03:59:37.245+0000] {task_command.py:416} INFO - Running <TaskInstance: All_Layer.clean_credit manual__2025-01-23T03:54:02.576089+00:00 [running]> on host 73fc507ad5e4
[2025-01-23T03:59:37.344+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='All_Layer' AIRFLOW_CTX_TASK_ID='clean_credit' AIRFLOW_CTX_EXECUTION_DATE='2025-01-23T03:54:02.576089+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-23T03:54:02.576089+00:00'
[2025-01-23T03:59:37.356+0000] {base.py:73} INFO - Using connection ID 'spark-conn' for task execution.
[2025-01-23T03:59:37.360+0000] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --conf spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension --conf spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog --jars /opt/***/jars/hadoop-aws-3.3.4.jar,/opt/***/jars/s3-2.18.41.jar,/opt/***/jars/aws-java-sdk-bundle-1.12.262.jar,/opt/***/jars/aws-java-sdk-1.12.367.jar,/opt/***/jars/delta-core_2.12-2.2.0.jar,/opt/***/jars/delta-storage-2.2.0.jar, --packages org.apache.hadoop:hadoop-aws:3.3.4 --num-executors 2 --total-executor-cores 2 --executor-cores 2 --executor-memory 2g --driver-memory 1g --name arrow-spark --deploy-mode client /opt/***/jobs/python/processing_credit.py s3a://lakehouse/bronze/credits.parquet s3a://lakehouse/silver/credit
[2025-01-23T03:59:37.543+0000] {spark_submit.py:579} INFO - /home/***/.local/lib/python3.9/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2025-01-23T03:59:41.905+0000] {spark_submit.py:579} INFO - :: loading settings :: url = jar:file:/home/***/.local/lib/python3.9/site-packages/pyspark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-01-23T03:59:42.165+0000] {spark_submit.py:579} INFO - Ivy Default Cache set to: /home/***/.ivy2/cache
[2025-01-23T03:59:42.166+0000] {spark_submit.py:579} INFO - The jars for the packages stored in: /home/***/.ivy2/jars
[2025-01-23T03:59:42.178+0000] {spark_submit.py:579} INFO - org.apache.hadoop#hadoop-aws added as a dependency
[2025-01-23T03:59:42.183+0000] {spark_submit.py:579} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-9647e1b1-d18a-465b-abb6-84761e8564ed;1.0
[2025-01-23T03:59:42.185+0000] {spark_submit.py:579} INFO - confs: [default]
[2025-01-23T03:59:42.708+0000] {spark_submit.py:579} INFO - found org.apache.hadoop#hadoop-aws;3.3.4 in spark-list
[2025-01-23T03:59:42.761+0000] {spark_submit.py:579} INFO - found com.amazonaws#aws-java-sdk-bundle;1.12.262 in central
[2025-01-23T03:59:42.852+0000] {spark_submit.py:579} INFO - found org.wildfly.openssl#wildfly-openssl;1.0.7.Final in central
[2025-01-23T03:59:42.922+0000] {spark_submit.py:579} INFO - :: resolution report :: resolve 699ms :: artifacts dl 38ms
[2025-01-23T03:59:42.927+0000] {spark_submit.py:579} INFO - :: modules in use:
[2025-01-23T03:59:42.930+0000] {spark_submit.py:579} INFO - com.amazonaws#aws-java-sdk-bundle;1.12.262 from central in [default]
[2025-01-23T03:59:42.930+0000] {spark_submit.py:579} INFO - org.apache.hadoop#hadoop-aws;3.3.4 from spark-list in [default]
[2025-01-23T03:59:42.931+0000] {spark_submit.py:579} INFO - org.wildfly.openssl#wildfly-openssl;1.0.7.Final from central in [default]
[2025-01-23T03:59:42.931+0000] {spark_submit.py:579} INFO - ---------------------------------------------------------------------
[2025-01-23T03:59:42.932+0000] {spark_submit.py:579} INFO - |                  |            modules            ||   artifacts   |
[2025-01-23T03:59:42.932+0000] {spark_submit.py:579} INFO - |       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-01-23T03:59:42.932+0000] {spark_submit.py:579} INFO - ---------------------------------------------------------------------
[2025-01-23T03:59:42.933+0000] {spark_submit.py:579} INFO - |      default     |   3   |   0   |   0   |   0   ||   3   |   0   |
[2025-01-23T03:59:42.933+0000] {spark_submit.py:579} INFO - ---------------------------------------------------------------------
[2025-01-23T03:59:42.934+0000] {spark_submit.py:579} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-9647e1b1-d18a-465b-abb6-84761e8564ed
[2025-01-23T03:59:42.937+0000] {spark_submit.py:579} INFO - confs: [default]
[2025-01-23T03:59:42.963+0000] {spark_submit.py:579} INFO - 0 artifacts copied, 3 already retrieved (0kB/28ms)
[2025-01-23T03:59:43.738+0000] {spark_submit.py:579} INFO - 25/01/23 03:59:43 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-01-23T03:59:44.826+0000] {spark_submit.py:579} INFO - File "/opt/***/jobs/python/processing_credit.py", line 36
[2025-01-23T03:59:44.827+0000] {spark_submit.py:579} INFO - .getOrCreate()
[2025-01-23T03:59:44.828+0000] {spark_submit.py:579} INFO - IndentationError: unexpected indent
[2025-01-23T03:59:44.852+0000] {spark_submit.py:579} INFO - 25/01/23 03:59:44 INFO ShutdownHookManager: Shutdown hook called
[2025-01-23T03:59:44.855+0000] {spark_submit.py:579} INFO - 25/01/23 03:59:44 INFO ShutdownHookManager: Deleting directory /tmp/spark-9cb10a60-b784-4a2f-a9ee-8a71423b4e11
[2025-01-23T03:59:44.915+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --conf spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension --conf spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog --jars /opt/***/jars/hadoop-aws-3.3.4.jar,/opt/***/jars/s3-2.18.41.jar,/opt/***/jars/aws-java-sdk-bundle-1.12.262.jar,/opt/***/jars/aws-java-sdk-1.12.367.jar,/opt/***/jars/delta-core_2.12-2.2.0.jar,/opt/***/jars/delta-storage-2.2.0.jar, --packages org.apache.hadoop:hadoop-aws:3.3.4 --num-executors 2 --total-executor-cores 2 --executor-cores 2 --executor-memory 2g --driver-memory 1g --name arrow-spark --deploy-mode client /opt/***/jobs/python/processing_credit.py s3a://lakehouse/bronze/credits.parquet s3a://lakehouse/silver/credit. Error code is: 1.
[2025-01-23T03:59:44.923+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=All_Layer, task_id=clean_credit, execution_date=20250123T035402, start_date=20250123T035937, end_date=20250123T035944
[2025-01-23T03:59:44.945+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 62 for task clean_credit (Cannot execute: spark-submit --master spark://spark-master:7077 --conf spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension --conf spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog --jars /opt/***/jars/hadoop-aws-3.3.4.jar,/opt/***/jars/s3-2.18.41.jar,/opt/***/jars/aws-java-sdk-bundle-1.12.262.jar,/opt/***/jars/aws-java-sdk-1.12.367.jar,/opt/***/jars/delta-core_2.12-2.2.0.jar,/opt/***/jars/delta-storage-2.2.0.jar, --packages org.apache.hadoop:hadoop-aws:3.3.4 --num-executors 2 --total-executor-cores 2 --executor-cores 2 --executor-memory 2g --driver-memory 1g --name arrow-spark --deploy-mode client /opt/***/jobs/python/processing_credit.py s3a://lakehouse/bronze/credits.parquet s3a://lakehouse/silver/credit. Error code is: 1.; 10629)
[2025-01-23T03:59:44.964+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-01-23T03:59:45.002+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
